test/potluck_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'minitest/autorun'
+require 'minitest/emoji'
+require './lib/dish'
+require './lib/potluck'
+
+class DishTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+  end
+
+  def test_it_holds_a_date
+    potluck = Potluck.new("7-13-18")
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_starts_without_dishes
+    potluck = Potluck.new("7-13-18")
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_adds_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck.add_dish(couscous_salad)
+    assert_instance_of Dish, potluck.dishes[0]
```

Comment:
This gets you there, but sometimes like to be more specific in the assertion since we can:

```
assert_equal [couscous_salad], potluck.dishes
```